src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -57,4 +57,18 @@ public boolean isValid() {
      * e.g. trim strings
      */
     public abstract void sanitizeForSaving();
+
+    /**
+     * Adds {@code error} to {@code errors} if {@code error} is a non-empty string.
+     *
+     * @param error An {@code error} message, possibly empty.
+     * @param errors A List of {@code errors}, to add {@code error} to.
```

Comment:
You did not follow this comment exactly. This comment makes a reference to one of the parameters.